packages/carbon-web-components/src/components/pagination/pagination.ts

```diff
@@ -345,8 +353,11 @@ class CDSPagination extends FocusMixin(HostListenerMixin(LitElement)) {
       .join(' ');
 
     return html`
-      <div class="${prefix}--pagination__left">
-        <label for="select" class="${prefix}--pagination__text"
+      <div part="left-container" class="${prefix}--pagination__left">
+        <label
+          part="pagination-text-left"
+          for="select"
+          class="${prefix}--pagination__text"
           ><slot name="label-text">${itemsPerPageText}</slot></label
         >
         <cds-select
```

Comment:
Missing `part` attributes on `<cds-select>` and `<cds-select-item>` elements, both part of the shadow root for this component.